src/rulesets/Base/arraymath.jl

```diff
@@ -73,6 +73,92 @@ function rrule(
 end
 
 
+#####
+##### `muladd`
+#####
+
+function rrule(
+        ::typeof(muladd),
+        A::AbstractMatrix{<:CommutativeMulNumber},
+        B::AbstractVecOrMat{<:CommutativeMulNumber},
+        z::Union{CommutativeMulNumber, AbstractVecOrMat{<:CommutativeMulNumber}},
+    )
+    # The useful case, mul! fused with +
+    function muladd_pullback_1(Ȳ)
```

Comment:
I think the convention is to just call it `f_pullback` for function `f` for all methods. Is there a particular benefit to adding the `_1` designation?

Reply:
If the purpose of naming them at all is to make stack traces more informative, then one more clue can’t hurt, I think.